src/libraries/System.Private.CoreLib/src/System/TimeSpan.cs

```diff
@@ -15,15 +15,15 @@ namespace System
     // to 100 nanoseconds. While this maps well into units of time such as hours
     // and days, any periods longer than that aren't representable in a nice fashion.
     // For instance, a month can be between 28 and 31 days, while a year
-    // can contain 365 or 366 days.  A decade can have between 1 and 3 leap-years,
+    // can contain 365 or 366 days.  A decade can have between 1 and 3 leap years,
     // depending on when you map the TimeSpan into the calendar.  This is why
     // we do not provide Years() or Months().
     //
     // Note: System.TimeSpan needs to interop with the WinRT structure
     // type Windows::Foundation:TimeSpan. These types are currently binary-compatible in
     // memory so no custom marshalling is required. If at any point the implementation
     // details of this type should change, or new fields added, we need to remember to add
-    // an appropriate custom ILMarshaler to keep WInRT interop scenarios enabled.
+    // an appropriate custom ILMarshaler to keep WinRT interop scenarios enabled.
```

Comment:
The whole comment starting with `Note:` can be deleted. It is no longer applicable. CsWinRT does not depend on the internal layout of System.TimeSpan  - it does proper marshalling for it https://github.com/microsoft/CsWinRT/blob/3d85a632afcf91bc44bf9e64f002aa920e7d435b/src/WinRT.Runtime/Projections/SystemTypes.cs#L25-L35.

cc @Sergio0694 @manodasanW

Reply:
Ah, yeah good point, this shouldn't be needed anymore. I wonder if there's any code out there though that's depending on this to pass `TimeSpan` values as blittable to some native method, such that changing this here could break that? Not sure it's necessarily a concern though given that it'd technically be undefined behavior I guess.